core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -228,40 +228,38 @@ <h3>Roles</h3>
           </div>
 
 
-          {% if can_modify_roles %}
-            <div ng-hide="isRolesFormOpen">
-              <button type="button" class="btn btn-default protractor-test-edit-roles" ng-click="openEditRolesForm()">
-                Edit Roles
-              </button>
-            </div>
+          <div ng-if="can_modify_roles" ng-hide="isRolesFormOpen">
```

Comment:
@shubha1593, Will it be good to use ng-if and ng-hide in same div? (Although it's correct)

Reply:
@DubeySandeep, I think it's okay to use them in same div.